ledger/participant-integration-api/src/main/scala/platform/apiserver/StandaloneApiServer.scala

```diff
@@ -145,7 +260,7 @@ final class StandaloneApiServer(
         config.tlsConfig,
         AuthorizationInterceptor(
           authService,
-          executionContext,
+          materializer.executionContext,
```

Comment:
We should protect the akka internal execution contexts and avoid any direct usages.

Reply:
yeah about that...(@SamirTalwar-DA don't kill me please) I not too fond of rolling a class all the time with the acquire, I think is nicer just composing the ResourceOwners.
Aaand thanks for pointing out because I was doing some digging, and it seems to me that this is really not a place for the Resoure's EC, since that is ticking in some runtime code for the AuthorizationInterceptor, so I switched it to the serviceExecutionContext. (@SamirTalwar-DA I saw you were operating in that area, can you check maybe whether my conclusion holds?)